Client.xcodeproj/xcshareddata/xcschemes/Fennec.xcscheme

```diff
@@ -140,8 +140,8 @@
    </BuildAction>
    <TestAction
       buildConfiguration = "Fennec"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
```

Comment:
Could we undo these changes?

Reply:
Not sure how this change got it, i will try to undo.